src/terraform/providers/terraform-provider-avere/resource_vfxt.go

```diff
@@ -1174,17 +1189,24 @@ func finishFixedQuotaPercent(corefilers []*CoreFiler, averevfxt *AvereVfxt) erro
 		if err := averevfxt.RemoveCustomSetting(QuotaCacheMoveMax); err != nil {
 			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
 		}
+		if err := averevfxt.RemoveCustomSetting(QuotaDivisorFloor); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
+		}
+		if err := averevfxt.RemoveCustomSetting(QuotaMaxMultiplierForInvalidatedMassQuota); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
```

Comment:
Error should print QuotaMaxMultiplierForInvalidatedMassQuota

Reply:
fixed